src/github.com/couchbase/sync_gateway/db/document.go

```diff
@@ -27,6 +27,7 @@ type syncData struct {
 	Flags           uint8               `json:"flags,omitempty"`
 	Sequence        uint64              `json:"sequence"`
 	UnusedSequences []uint64            `json:"unused_sequences,omitempty"`
+	RecentSequences []uint64            `json:"recent_sequences,omitempty"`
```

Comment:
Can you add docs for this field?  I realize it won't "match" the other undocumented fields, but I think the ugliness is worth it from the perspective of someone trying to understand this struct.  Actually these docs that were elsewhere in the code look perfect:

> The server TAP/DCP feed will deduplicate multiple revisions for the same doc if they occur in the same mutation queue processing window. This results in missing sequences on the change listener.  To account for this, we track the recent sequence numbers for the document.